packages/stores/src/mockStore.ts

```diff
@@ -213,6 +214,7 @@ const mock = (): TRootStore => {
             current_language: 'EN',
             is_network_online: false,
             is_language_changing: false,
+            server_time: moment(),
```

Comment:
@shafin-deriv Same comment as [here](https://github.com/binary-com/deriv-app/pull/7300#discussion_r1163774489) 👀

Reply:
Made the server_time optional for now